packages/frontend/src/components/dialogs/ConfirmDeleteMessage.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+
+import Dialog, {
+  DialogBody,
+  DialogContent,
+  DialogFooter,
+  FooterActions,
+} from '../Dialog'
+import FooterActionButton from '../Dialog/FooterActionButton'
+import useTranslationFunction from '../../hooks/useTranslationFunction'
+
+import type { DialogProps } from '../../contexts/DialogContext'
+import { BackendRemote, Type } from '../../backend-com'
+import { C } from '@deltachat/jsonrpc-client'
+
+export type Props = {
+  accountId: number
+  msg: Type.Message
+  canSend: boolean
+} & DialogProps
+
+export default function ConfirmDeleteMessageDialog(props: Props) {
+  const tx = useTranslationFunction()
+
+  const { accountId, msg, onClose } = props
+
+  const deleteForAllPossible =
+    msg.sender.id === C.DC_CONTACT_ID_SELF && props.canSend
+
+  const deleteMessage = (deleteForEveryone: boolean) => {
+    if (deleteForEveryone) {
+      BackendRemote.rpc.deleteMessagesForAll(accountId, [msg.id])
```

Comment:
This seems like a place where it is important to report an error should one happen.

Though I know that the presence of the error is otherwise visible to the user via the fact that the message didn't disappear from the chat.

Reply:
I added a userFeedback for errors here.

@r10s 
I think it would make sense to add at least a confirmation for "delete for all" but then we need a new translation for that